homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -63,7 +63,6 @@
         vol.Required(CONF_ORIGIN): cv.string,
         vol.Required(CONF_DESTINATION): cv.string,
         vol.Required(CONF_REGION): vol.In(REGIONS),
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Comment:
Let's keep the name option in the platform schema so we can import legacy configuration without problems.